src/tools/illink/test/Mono.Linker.Tests/TestCasesRunner/ResultChecker.cs

```diff
@@ -49,13 +49,43 @@ public ResultChecker ()
 			_linkedReaderParameters = linkedReaderParameters;
 		}
 
-		static void VerifyIL (NPath pathToAssembly)
+		static void VerifyIL (NPath pathToAssembly, AssemblyDefinition linked)
 		{
 			var verifier = new ILVerifier (pathToAssembly);
-			foreach (var result in verifier.Results) {
-				if (result.Code == ILVerify.VerifierError.None)
-					continue;
+			foreach (var result in verifier.Results)
 				Assert.Fail (ILVerifier.GetErrorMessage (result));
+		}
+
+		static void ValidateTypeRefsHaveValidAssemblyRefs (AssemblyDefinition linked)
+		{
+			Console.WriteLine ("Validating type references have valid assembly references: " + linked.FullName);
```

Comment:
Where does this go?
Just curious - I haven't seen us using Console.WriteLine in tests for anything yet.